KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.java

```diff
@@ -139,6 +150,110 @@ private void loadTiles() {
             tiles.add(newTile);
         }
     }
+
+    /**
+     * Create the board's node network of settlements and roads
+     */
+    public final void loadNodes() {
+        
+        // Declare node attribute arrays
+        // Settlement attribute arrayLists
+        int settlementX;
+        int settlementY;
+        int settlementRoad1[] = new int[54];
```

Comment:
why is the size 54?

Reply:
What is mean is what is this for?

Reply:
there are 54 settlement nodes in the map
those hold data to be used to link the nodes. each of the integers is the index of the road to connect to